packages/components/src/RadarChart/RadarChart.component.js

```diff
@@ -136,10 +136,47 @@ Dot.propTypes = {
 	index: PropTypes.number,
 };
 
+/**
+ * This function provides a clickable axis dot element
+ * @param {Object} props the current props of the Radar
+ */
+// TODO 6.0: remove this export
+export function DotWithClick(props) {
+	const { activeAxis, fill, index, ...rest } = props;
+	let radius = 2;
+	let strokeRadius = 12;
```

Comment:
I would prefer consts (such as `DEFAULT_RADIUS`, `ACTIVE_RADIUS`, etc.), but up to you.

Reply:
Now using const object for state params.